hackathon/react-native-heroes/.vercel/project.json

```diff
@@ -0,0 +1 @@
+{"projectId":"prj_BI3naXEoeM0E4VP9uUdmE41rfeLb","orgId":"DiQ5Nk2nsd1fNB9njlWjMsW4"}
```

Comment:
Your project credentials are leaked to the world

Delete it and create a new one OR reset the keys